virttest/utils_misc.py

```diff
@@ -2107,16 +2107,17 @@ def get_cpu_vendor(cpu_info="", verbose=True):
 
 def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
     """
-    Get the machine type the host support,return a list of machine type
+    Get the machine type the host support,return a tuple (s, c),
+    s is machine type list, and c is system version list
     """
     o = process.system_output("%s -M ?" % qemu_binary).splitlines()
     s = []
     c = []
     for item in o[1:]:
-        machine_list = re.split(r"\s+", item, 1)
-        if machine_list[0] != "none":
+        if not ("alias" in item or "none" in item):
+            machine_list = re.split(r"\s+", item, 1)
             s.append(machine_list[0])
-            c.append(machine_list[1])
+            c.append(re.sub(" \(default\)", "", machine_list[1]))
```

Comment:
this line is unnecessary, right?

Reply:
No, add this on purpose.
```
# /usr/libexec/qemu-kvm -M ?
Supported machines are:
pc                   RHEL 7.5.0 PC (i440FX + PIIX, 1996) (alias of pc-i440fx-rhel7.5.0)
pc-i440fx-rhel7.5.0  RHEL 7.5.0 PC (i440FX + PIIX, 1996) (default)
pc-i440fx-rhel7.4.0  RHEL 7.4.0 PC (i440FX + PIIX, 1996)
```
In the vm, with command "dmidecode", system version is  `RHEL 7.5.0 PC (i440FX + PIIX, 1996)` , not `RHEL 7.5.0 PC (i440FX + PIIX, 1996) (default)`.  we need to strip `(default)` .